source/Ship.cpp

```diff
@@ -2676,32 +2676,20 @@ int Ship::Scan(const PlayerInfo &player)
 	bool activeScanning = false;
 	int result = 0;
 	auto doScan = [&distanceSquared, &startedScanning, &activeScanning, &result]
-			(double &elapsed, const double speed, const double scannerRange,
+			(double &elapsed, const double speed, const double scannerRangeSquared,
 					const double depth, const int event)
 	-> void
 	{
-		if(elapsed < SCAN_TIME && distanceSquared < scannerRange)
+		const int MAX_RANGE_FACTOR = 3 * 3; // three standard deviations AKA triple range
```

Comment:
this is one math operation too much here(micro optimize)

also move a constant like this into the(or an) unnamed namespace.

Reply:
It is a constant, so it is calculated at runtime. The `3 * 3` is necessary for readability, so the readable sees the important information: 3 times the distance.

Reply:
runtime would be strange XD

still move it to an unnamed namespace for style

Reply:
It is sitting next to SCAN_TIME now.